### ISSUE TITLE
MultiSelect: 'Scrollable region must have keyboard access' accessibility violation

### ISSUE THREAD

**Issue by elena-ung** [primefaces/primeng#18184]
Install the Axe Dev Tools for Accessibility plug-in.
Navigate to https://v17.primeng.org/multiselect on Chrome browser
Scroll to Template section and open the multiselect control
Scan the page with Axe

Observe 'Scrollable region must have keyboard access' accessibility violation

![Image](https://github.com/user-attachments/assets/c5562dca-531a-4038-8a5a-22d27c0971d4)

Angular version: 18.2.9
PrimeNG version: v17
Node version: 18.19.1
